exporter/opentelemetry-exporter-zipkin-proto/src/opentelemetry/exporter/zipkin/proto/node_endpoint.py

```diff
@@ -0,0 +1,85 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Zipkin Exporter Endpoints"""
+
+import ipaddress
+from typing import Optional, Union
+
+from opentelemetry import trace
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+
+IpInput = Union[str, int, None]
+
+
+class NodeEndpoint:
```

Comment:
This class looks the same as the one in the JSON exporter `opentelemetry.exporter.zipkin.node_endpoint.NodeEndpoint`, can we just reuse it? I see we are using the existing one in the file below (exporter/opentelemetry-exporter-zipkin-proto/tests/encoder/test_v2_protobuf.py)

Reply:
@aabmass thanks for catching this, i thought i'd removed it already